pkg/monitoring/vms/prometheus/fakeCollector.go

```diff
@@ -0,0 +1,64 @@
+package prometheus
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	libvirt "libvirt.org/libvirt-go"
+
+	k6tv1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/stats"
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/statsconv"
+	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/statsconv/util"
+)
+
+type fakeCollector struct {
```

Comment:
instead of using the word `fake` or `test` for different things maybe it makes sense to converage on the word `stub` instead for both?

Reply:
I am not sure what you mean. I don't use `test` as far as I know